service/src/main/java/com/derpgroup/derpwizard/manager/AbstractManager.java

```diff
@@ -58,7 +53,8 @@ public AbstractManager(){
    *          The document builder to append messages to, not null
    */
   public void handleRequest(@NonNull VoiceInput voiceInput, @NonNull SsmlDocumentBuilder builder) {
-    appendToConversationHistory(voiceInput);
+    CommonMetadata metadata = voiceInput.getMetadata();
+    ConversationHistoryUtils.appendToConversationHistory(voiceInput, metadata, mapper);
```

Comment:
Why are we extracting CommonMetadata from VoiceInput if we're turning around and passing both into this function right here.